core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -4672,72 +4672,134 @@ def fresh_show(self):
+                    if group1 is None and group2 is None:
+                        self.groups = []
+                    else:
+                        self.groups = [group1, group2]
+                        self.groups.sort(key=lambda g: g["group_id"])
+                counter = (counter + 2) // 3
+            else:
+                if counter == 1:
+                    self.groups = [
+                        {"group_id": valid_fps[0], "indexes": [valid_fps[0]]}
+                    ]
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)